recipes/pattern/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set name = "pattern" %}
+{% set version = "3.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/P/Pattern/Pattern-{{ version }}.tar.gz
+  sha256: 1ced3336a5189e3025ece970229126bc7e6adf2e73a91c2e625b3a45eb0720d9
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [win]
```

Comment:
You can remove this and add `noarch: python`

Reply:
@isuruf i believe there are some modules that make this package incompatible with windows so i don't think it can be noarch for now. please review my changes, thank you.

Reply:
Modules inside the package or dependencies?  Is it is dependencies that is fine to be noarch, it just won't be installable. Also, the website says it works on windows: https://www.clips.uantwerpen.be/pages/pattern

Reply:
Modules inside the package. I believe the website has not been updated to reflect the new version 3.6.0. Specifically, I was getting errors during Windows tests when importing `pattern.server` and `pattern.vector.svm`. I believe `pattern.server` is not working because it relies on `pwd` and `grp`.

Reply:
I don't think that is enough of a reson to not have a windows package.  There are often platform-specific bugs. It seems like there is a lot of other functionality that *does* work on windows

Reply:
you're right, sorry for the confusion.